### PR TITLE
Add example of rpm build option override

### PR DESCRIPTION
**Pull request by simaishi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,6 +359,15 @@ For example, if you want to build RPMs using `test/rpm_build` image and override
 RPM_BUILD_IMAGE=test/rpm_build RPM_BUILD_OPTIONS=my_options bin/build -d images -r manageiq -b
 ```
 
+An example options.yml to use "feature" branch on the "example" fork of ManageIQ/manageiq:
+
+```
+repos:
+  manageiq:
+    url:  https://github.com/example/manageiq.git
+    ref:  feature
```
Comment (jrafanie) [inline README.md:368]: For reference, I tried a git ref, a sha1, here and it doesn't work.  It needs to be a branch.  I think a tag might also fail.

**Comment by jrafanie** [comment]
It's worth mentioning that you must specify a relative path to the DIRECTORY of the options.yml and clearly, it must be named options.yml.

If you try a FQ path:  

```
RPM_BUILD_OPTIONS="/Users/joerafaniello/Code/manageiq-pods" bin/build -d images -b -r jrafanie -p -t joe_latest6
```

You get:

```
 docker run -it --rm -v /Users/joerafaniello/Code/manageiq-pods/images/manageiq-base/rpms:/root/BUILD/rpms -v /Users/joerafaniello/Code/manageiq-pods//Users/joerafaniello/Code/manageiq-pods:/root/OPTIONS manageiq/rpm_build:latest build
```

It fails to find the options.yml and uses defaults.

File: README.md (hunk 1 above)
Comment (simaishi) [reply README.md]: Added `must be tag or branch`. We use tag for release builds, so tag works 😄

**Comment by simaishi** [comment]
Added a note about RPM_BUILD_OPTIONS.

**Review by jrafanie — APPROVED** [review]
LGTM

**Pull request merged**